Microsoft.ML.FairLearn/api-example.cs

```diff
@@ -0,0 +1,52 @@
+// See https://aka.ms/new-console-template for more information
+using Microsoft.ML;
+using Microsoft.ML.AutoML;
+using Microsoft.ML.FairLearn;
```

Comment:
I've seen it spelled like this a few times (never by the people who actually created it), but Fairlearn is spelled in lowercase and we'd love to keep it that way. 🙂